src/view/main.js

```diff
@@ -83,9 +82,6 @@ class MainView extends Component {
         {route === 'RestoreSeed' && (
           <RestoreSeed store={store} wallet={wallet} />
         )}
-        {route === 'RestorePassword' && (
-          <RestorePassword store={store} wallet={wallet} nav={nav} />
-        )}
```

Comment:
Nice 👍So we're using the same set password screens as during the "generate new wallet" setup now?

Reply:
yes indeed :)